src/main.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# we don't use set -e, it messes up hide -F and reveal -F error handling
+set -e
```

Comment:
I had previously removed `set -e` from `src/main.sh`, because otherwise issues with (for example) expired keys would cause the script to stop here.

I've re-enabled `set -e` here, and disabled the `set -e` feature selectively, just around the `gpg` use I want to capture issues with. (You can see the changes in `commands/git_secret_hide.sh` and `src/_utils/_git_secret_tools.sh`.)

Is there a better way to do this?

Reply:
interested to hear your thoughts @sobolevn